src/xdp-utils.c

```diff
@@ -2349,3 +2349,67 @@ xdp_validate_serialized_icon (GVariant  *v,
 
   return TRUE;
 }
+
+#if G_ENCODE_VERSION (GLIB_MAJOR_VERSION, GLIB_MINOR_VERSION) >= G_ENCODE_VERSION (2, 66)
+gchar *
+xdp_transform_remote_uri_into_local (const gchar *uri)
+{
+  gchar *pathtext = NULL;
```

Comment:
Use `g_autofree gchar *` to remove the free calls below.

Reply:
Done with pathtext and fuse_mountpoint, and used g_autoptr for fileuri and gvfs_folder.